cmd/kube-apiserver/app/server.go

```diff
@@ -410,11 +420,12 @@ func (s *APIServer) Run(_ []string) error {
 	}
 
 	if secureLocation != "" {
+		timeout := time.Duration(MinTimeoutSecs+rand.Intn(MaxTimeoutSecs-MinTimeoutSecs)) * time.Second
```

Comment:
don't we want the same timeout for all http servers within one Kubelet?
I think it definitely makes sense to differentiate it between Kubelets, but maybe within a Kubelet they should be the same?